data/records/delphi-conditions-database.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "abstract": {
+      "description": "The DELPHI conditions database."
+    },
+    "accelerator": "CERN-LEP",
+    "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "80050"
+    },
+    "collections": [
+      "DELPHI"
+    ],
+    "date_created": [
+      "2000"
+    ],
+    "date_published": "2025",
+    "distribution": {
+      "formats": [
+        "zebra"
+      ],
+      "number_files": 7,
+      "size": 3205865472
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:18ab9196",
+        "size": 1497251840,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/condition-data/DBcalb.dat"
+      },
+      {
+        "checksum": "adler32:b74dd263",
+        "size": 32432128,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/condition-data/DBgeom.dat"
+      },
+      {
+        "checksum": "adler32:21bf3c40",
+        "size": 1655250944,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/condition-data/DBlepm.dat"
+      },
+      {
+        "checksum": "adler32:dc088849",
+        "size": 40960,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/condition-data/DBmisc.dat"
+      },
+      {
+        "checksum": "adler32:0190cfad",
+        "size": 20307968,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/condition-data/DBrunt.dat"
+      },
+      {
+        "checksum": "adler32:0fd5e6e0",
+        "size": 450560,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/condition-data/DBscon.dat"
+      },
+      {
+        "checksum": "adler32:1d8e2a3b",
+        "size": 131072,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/condition-data/DBsysf.dat"
+      }
+    ],
+    "license": {
+      "attribution": "CC0-1.0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "80509",
```

Comment:
If you'd like to give it a DOI, we can use `10.7483/OPENDATA.DELPHI.18JY.GL8K`.

(Assuming one condition data record aggregating all the DAT files is enough. Alternatively, if you'd like to have distinct DOIs for distinct files, to be able to reference them independently, we could also create independent records for each DBsysf etc entity.)